src/Raven.Server/Web/System/AdminDatabasesHandler.cs

```diff
@@ -164,6 +156,28 @@ public Task Get()
             }
         }
 
+        private static void AddNode(ClusterTopology clusterTopology, DatabaseTopology topology, string name, string node)
+        {
+            if (topology.AllNodes.Count() == 0)
```

Comment:
Under what condition this can happen?
Only when the db is under delete, no?
And in that case, we should refuse adding to it.